### ISSUE TITLE
Neuron View Inconsistency 

### ISSUE THREAD

**Issue by mojivalipour** [jessevig/bertviz#32]
As you can see in the following example, the neuron view output is different than what you've provided in the readme of this repo. Please also check this one. 

The current output: 
![image](https://user-images.githubusercontent.com/59541910/72221756-cb34cb00-3572-11ea-9ee5-17641053698a.png)

The demanded output:
![image](https://user-images.githubusercontent.com/59541910/72221789-1bac2880-3573-11ea-837b-b7439c446d32.png)

**Comment by jessevig** [comment]
Can you show me how you are importing "show"? It should be imported as follows:
from bertviz.neuron_view import show

**Comment by jessevig** [comment]
Okay, I see. You actually need to click the plus sign that comes up when you hover over the tokens on the left.

**Comment by mojivalipour** [comment]
Cool, this is a nice feature, but why you don't integrate all of these things into one? It's really a useful library for attention models, but there are some integration and documentation problems in the current version, which make it unsuitable for mass usage.

**Comment by jessevig** [comment]
Thanks. I'm working on expanding the documentation.Do you have a specific suggestion for a different integration? The reason for the difference between the neuron view and the other views is that the neuron view needs access to the queries and keys, which are not available in the current version of transformers. I don't want to have the head view and model view rely on the forked version of transformers because this creates a dependency that needs to be updated with each new release of transformer models.

**Comment by mojivalipour** [comment]
Yes, I see why you did this, but I think HuggingFace (@huggingface) are friendly if you raise an issue and inform them about your technology. I think it is good for them to use and integrate your repo into theirs. Their repo, which is based on Transformers, suffered from the lack of this kind of visualization support. This solution is the best idea if they (@clmnt, @julien-c, @LysandreJik, @n1t0, @Pierrci, @thomwolf, and @VictorSanh) cooperate.
